setup/symfony_server.rst

```diff
@@ -244,6 +244,105 @@ server provides a ``run`` command to wrap them as follows:
     # stop the web server (and all the associated commands) when you are finished
     $ symfony server:stop
 
+Docker Integration
+------------------
+
+The local Symfony server provides full `Docker`_ integration for projects that
+use it. First, make sure to expose the container ports:
+
+.. code-block:: yaml
+
+    # docker-compose.override.yaml
+    services:
+        database:
+            ports:
+                - "3600"
+
+        redis:
+            ports:
+                - "6379"
+
+        # ...
+
+Then, check your service names and update them if needed (Symfony creates
+environment variables following the name of the services so they can be
+autoconfigured):
+
+.. code-block:: yaml
+
+    # docker-compose.yaml
+    services:
+        # DATABASE_URL
+        database: ...
+        # MONGODB_DATABASE, MONGODB_SERVER
+        mongodb: ...
+        # REDIS_URL
+        redis: ...
+        # ELASTISEARCH_HOST, ELASTICSEARCH_PORT
+        elasticsearch: ...
+        # RABBITMQ_DSN
+        rabbitmq: ...
+
+If you can't or don't want to update the service names, you must update the
+Symfony application configuration to use the new environment variables. For
+example, if you want to keep a service called ``mysql`` instead of renaming it
+to ``database``, the env var will be called ``MYSQL_URL`` instead of
+``DATABASE_URL``, so you must update the configuration as follows:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/doctrine.yaml
+        doctrine:
+            dbal:
+                url: '%env(MYSQL_URL)%'
```

Comment:
I didn't try it but I think defining `MYSQL_URL=${DATABASE_URL}` in `.env.local`  should work and seems less disruptive for the application configuration. Do you think it would be better that way?

Reply:
Much better indeed. I just added that. Please review it! Thanks.